js/tests/unit/util/scrollbar.spec.js

```diff
@@ -180,5 +182,67 @@ describe('ScrollBar', () => {
 
       Scrollbar.reset()
     })
+
+    describe('Body Handling', () => {
+      it('should hide scrollbar and reset it to its initial value', () => {
+        const el = document.body
+        const inlineStylePadding = '10px'
+        el.style.paddingRight = inlineStylePadding
+
+        const styleSheetPadding = '7px'
+        appendCss(`body { padding-right: ${styleSheetPadding} }`)
+        const originalPadding = getRightPadding(el)
+        expect(originalPadding).toEqual(parseInt(inlineStylePadding)) // Respect only the inline style as it has prevails this of css
+        const originalOverFlow = 'auto'
+        el.style.overflow = originalOverFlow
```

Comment:
thank you for pointing it out.

I have handled it like this
```js


  beforeAll(() => {
    fixtureEl = getFixture()
    // custom fixture to avoid extreme style values
    fixtureEl.removeAttribute('style')
  })

```